rpcs3/rpcs3qt/pad_led_settings_dialog.cpp

```diff
@@ -0,0 +1,90 @@
+﻿#include "pad_led_settings_dialog.h"
+#include "pad_settings_dialog.h"
+#include <QColorDialog>
+
+pad_led_settings_dialog::pad_led_settings_dialog(const int& colorR, const int& colorG, const int& colorB, const bool& led_low_battery_blink, const bool& led_battery_indicator, const int& led_battery_indicator_brightness, QDialog * parent)
+    : QDialog(parent)
+    , ui(new Ui::pad_led_settings_dialog)
+    , m_cR(colorR)
+    , m_cG(colorG)
+    , m_cB(colorB)
+    , m_low_battery_blink(led_low_battery_blink)
+    , m_battery_indicator(led_battery_indicator)
+    , m_battery_indicator_brightness(led_battery_indicator_brightness)
+{
+	ui->setupUi(this);
+	setFixedSize(size());
+
+	// Fill the form
+	ui->hs_indicator_brightness->setValue(led_battery_indicator_brightness);
+	ui->cb_led_blink->setChecked(led_low_battery_blink);
+	ui->cb_led_indicate->setChecked(led_battery_indicator);
+	switch_groupboxes(m_battery_indicator);
+	connect(ui->b_close, &QAbstractButton::clicked, this, &pad_led_settings_dialog::accept);
+	connect(ui->b_apply, &QAbstractButton::clicked, this, &pad_led_settings_dialog::pass_led_settings_signal);
+	connect(this, &pad_led_settings_dialog::pass_led_settings, qobject_cast<pad_settings_dialog*>(parent), &pad_settings_dialog::apply_led_settings);
```

Comment:
I thought I recommended to put this into pad_settings_dialog

Reply:
I must've missed it, the last connect is indeed safer in the `pad_settings_dialog`, thank you. The rest of them are to stay in `pad_led_settings_dialog`, correct?

Reply:
yes